docs/source/AdministratorGuide/Systems/DataManagement/fts3.rst

```diff
@@ -44,9 +44,9 @@ Enable FTS transfers in the RMS
 
 In order for the transfers to be submitted to the FTS system:
 
-   * `Systems/RequestManagementSystem/Agents/RequestExecutingAgent/OperationHandlers/ReplicateAndRegister/FTSMode` must be True
-   * `Systems/RequestManagementSystem/Agents/RequestExecutingAgent/OperationHandlers/ReplicateAndRegister/FTSBannedGroups` should contain the list of groups for which you'd rather do direct transfers.
-   * `Systems/RequestManagementSystem/Agents/RequestExecutingAgent/OperationHandlers/ReplicateAndRegister/UseNewFTS3` should be True in order to use this new FTS system (soon to be deprecated)
+   * ``Systems/RequestManagementSystem/Agents/RequestExecutingAgent/OperationHandlers/ReplicateAndRegister/FTSMode`` must be True
```

Comment:
Maybe one can write:
```
In order for the transfers to be submitted to the FTS system, the following options in the configuration section ``Systems/RequestManagement/Agents/RequestExecutingAgent/OperationHandlers/ReplicateAndRegister/`` need to be set

* ``FTSMode = True`` ...
* ``FTSBannedGroups`` ...
* ``UseNewFTS3=True`` ... 

```

Of course this section path misses the `<Setup>` "folder" after RequestManagement (Not RequestManagementSystem). So maybe this can be shortened to just refer to the sub-section of the RequestExecutingAgent?

Reply:
Good idea !